src/toolbox/toolbox.expose.js

```diff
@@ -62,15 +62,22 @@
 		if (fn) { return fn.call($.mask); }
 	}
 	
-	var mask, exposed, loaded, config, overlayIndex;		
+	var mask, exposed, loaded, config, overlayIndex,
+		closingNow = false;		
```

Comment:
Nabd but in future just change this to be multivar declarations.

i.e.
```
var mask;
var exposed;
```

etc